### PR TITLE
Add validation

### PR DESCRIPTION
**Pull request by farouqdaif1**
- Add script to Html 
- Make the function

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -298,11 +298,11 @@ <h2>About me</h2>
...
-          <form class="form" action="https://formspree.io/f/mgerdpob" method="POST">
-            <input type="text" placeholder="Full name" required maxlength="30" class="name-email" name="name" id="full-name" >         
-            <input type="text" placeholder="First name" required maxlength="30" class="name-email" name="name" id="first-name" > 
-            <input type="text" placeholder="Last name" required maxlength="30" class="name-email" name="name" id="last-name"> 
+          <form class="form" action="https://formspree.io/f/mgerdpob" method="POST" id="form">
+            <input type="text" placeholder="Full name" required maxlength="30" class="name-email" name="namefULL" id="full-name" >         
             <input type="email" placeholder="Email address" required maxlength="30" class="name-email" name="email" id="email">
+            <small id="massage" ></small>
+
             <textarea rows="5" cols="10" required maxlength = "500" class="text"  name="textarea">Enter text here </textarea>
```
Comment (GzimAsani) [inline index.html]: - [ ] Add the text in `textarea` as a placeholder instead of a text

File: index.html [hunk 2]
```diff
@@ -298,11 +298,11 @@ <h2>About me</h2>
...
               so if you'd like to chat please get in</h4>
           </div>
-          <form class="form" action="https://formspree.io/f/mgerdpob" method="POST">
-            <input type="text" placeholder="Full name" required maxlength="30" class="name-email" name="name" id="full-name" >         
-            <input type="text" placeholder="First name" required maxlength="30" class="name-email" name="name" id="first-name" > 
-            <input type="text" placeholder="Last name" required maxlength="30" class="name-email" name="name" id="last-name"> 
+          <form class="form" action="https://formspree.io/f/mgerdpob" method="POST" id="form">
+            <input type="text" placeholder="Full name" required maxlength="30" class="name-email" name="namefULL" id="full-name" >         
             <input type="email" placeholder="Email address" required maxlength="30" class="name-email" name="email" id="email">
+            <small id="massage" ></small>
```
Comment (GzimAsani) [inline index.html]: - [ ] The error message should show up close to the "submit" button.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @farouqdaif1    
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good use of gitflow :+1: 

## Required Changes ♻️
- [ ] Be more descriptive in your pull request
- [ ] _Check the comments under the review for more information._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — APPROVED** [review]
Hi @farouqdaif1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**